activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -452,9 +458,11 @@ def schema_file_type(format = ActiveRecord.schema_format)
       end
       deprecate :schema_file_type
 
-      def schema_dump_path(db_config, format = ActiveRecord.schema_format)
+      def schema_dump_path(db_config, format = nil)
         return ENV["SCHEMA"] if ENV["SCHEMA"]
 
+        ActiveSupport::Deprecation.warn("`format` will be removed as a parameter in future versions; it should be included in the passed `db_config`") if format
```

Comment:
Let's make sure all the deprecations say the same thing.

```suggestion
        ActiveSupport::Deprecation.warn("`format` will be removed as a parameter in 7.2; it should be included in the database configuration.") if format
```